docs/zkapps/advanced-snarkyjs/merkle-tree.mdx

```diff
@@ -1,6 +1,14 @@
 ---
 title: Merkle Tree
 hide_title: true
+description: A comprehensive guide on how to use Merkle Trees to reference off-chain data in zkApps on Mina. Understand its algorithm, implementation, and significance in managing large data.
```

Comment:
who is "its" ? the Mina algorithm? or the Merkle tree ?

```suggestion
description: A comprehensive guide on how to use Merkle Trees to reference off-chain data in zkApps on Mina. Understand the Mina algorithm, implementation, and significance in managing large data.
```

Reply:
"its", in this case, refers to the Merkle Tree algorithm! I'll make that change.